src/shared/lib/datetime.js

```diff
@@ -1,106 +1,146 @@
-/*
-  Get a date object offset for the current timezone
-*/
-export const getDate = function() {
-  const date = new Date();
-  const utcDate = new Date(date.toUTCString());
-  utcDate.setHours(utcDate.getHours() - 7);
-  return new Date(utcDate);
-};
+import { DateTimeFormatter, ZonedDateTime, LocalDateTime, LocalDate, ZoneId } from '@js-joda/core';
+import '@js-joda/timezone/dist/js-joda-timezone-10-year-range'; // minimize package size by only importing tz data for current year ±5 yrs
 
-/*
-  Get date formatted in YYYY-M-D
-*/
-export const getYYYYMD = function(date = getDate(), sep = '-') {
-  let localDate = date;
-  if (typeof localDate === 'string') {
-    localDate = new Date(localDate);
-  }
-  const month = localDate.getUTCMonth() + 1;
-  const day = localDate.getUTCDate();
-  const year = localDate.getUTCFullYear();
+// util functions
+
+const currentJsDate = () => new Date(Date.now()); // allows us to mock current date
```

Comment:
I recommend getting the `.now()` at UTC timezone.

Reply:
> allows us to mock current date

Or is this only used for testing? Can we put testing related functions to a test prefixed file?